AnalysisCI/analyze_ci.py

```diff
@@ -0,0 +1,89 @@
+import os
+import sys
+import subprocess
+from GitWikiFormatter import GitWikiFormatter
+from GitWikiFormatter import GitApiHelper as Git
+from TravisEnvirExtractor import TravisEnvirExtractor
+
+previous_line = ""
+current_line = ""
+last_task_executed = ""
+temp_line = ""
+
+error_keywords = ["fail", "error"]
+success_keywords = ["success", "pass", "complete"]
+ignore_keywords = ["> task", "> transform", "w:"]
+TASK_STRING_INDEX = 0
+information_keywords = ["executed"]
+
+ignore_line = []
+error_lines = []
+success_lines = []
+information_lines = []
+
+for line in sys.stdin:
+
+    # Prevents infinite looping over the same input
+    previous_line = current_line
+    temp_line = line.strip("\n")
+    current_line = line.lower().strip("\n")
+
+    if current_line == previous_line:
```

Comment:
Is it possible that this will happen?

Reply:
yeah i was encountering this before, if there was no new value, it would keep spitting out the same line over and over